docs/docs/external/getting-started/guide.md

```diff
@@ -0,0 +1,28 @@
+- After you run these tests and get a flavor of how the system and protocols work, you are ready to move on to "real" networks, enhancing them with interoperan capabilities by incorporating Weaver into them. Check out the guidelines and templates for [Fabric](./enabling-weaver-network/fabric.md), [Corda](./enabling-weaver-network/corda.md), and [Besu](./enabling-weaver-network/besu.md) networks.
+
+If you wish to go further and understand Weaver specifics, dig into the code, or contribute to the open source project, check out the [project repository](https://github.com/hyperledger-labs/weaver-dlt-interoperability). For specific information about individual Weaver components, see:
+- [Relay](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/core/relay/README.md) module
+- [Fabric](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/core/drivers/fabric-driver/readme.md) and [Corda](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/core/drivers/corda-driver/README.md) drivers
+- [Fabric Interoperation Chaincode](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/core/network/fabric-interop-cc/README.md) and [Interoperation CorDapp](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/core/network/corda-interop-app/README.md)
+- Common [protobufs](https://github.com/hyperledger-labs/weaver-dlt-interoperability/tree/main/common/protos): compiled in [JavaScript](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/common/protos-js/README.md), [Golang](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/common/protos-go/README.md), and [Java](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/common/protos-java-kt/README.md)
+- Fabric Interoperation SDK in [Node.js](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/sdks/fabric/interoperation-node-sdk/README.md) and [Golang](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/sdks/fabric/go-sdk/readme.md)
+- Sample [Fabric](https://github.com/hyperledger-labs/weaver-dlt-interoperability/tree/main/samples/fabric) and [Corda](https://github.com/hyperledger-labs/weaver-dlt-interoperability/tree/main/samples/corda) applications for experimentation and testing
+- [Fabric](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/tests/network-setups/fabric/dev/README.md), [Corda](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/tests/network-setups/corda/README.md), and [Besu](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/tests/network-setups/besu/README.md) test network setups
+The Weaver [RFCs](https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/rfcs/README.md) contain detailed specifications of the models, data structures, protocols, and message formats.
```

Comment:
The Weaver -> - The Weaver

Reply:
Fabric interoperation SDK -> Fabric interoperation SDKs

Reply:
RFCs are not meant to be in the list, which is only for code modules.

Reply:
![image](https://user-images.githubusercontent.com/504640/129395753-bc11e15a-6c87-4569-88b5-1f183d4ca3e4.png)
@VRamakrishna I believe that it can't be continuing in that bullet point. Either it need to be a new member in the list or a new paragraph.